pytensor/tensor/slinalg.py

```diff
@@ -129,8 +142,10 @@ def conjugate_solve_triangular(outer, inner):
             return [grad]
 
 
-def cholesky(x, lower=True, on_error="raise"):
-    return Blockwise(Cholesky(lower=lower, on_error=on_error))(x)
+def cholesky(x, lower=True, on_error="raise", overwrite_a=False):
```

Comment:
```suggestion
def cholesky(x, lower=True, on_error="raise",):
```

No reason to provide that to users, We don't want them to create inplace Ops directly.

Instead you can mention it's done automatically by rewrites when appropriate

Reply:
Jax ignores it so the API is the same. We could also do that? https://jax.readthedocs.io/en/latest/_modules/jax/_src/scipy/linalg.html#cholesky

Reply:
if we don't allow users to pass it in, how does it get passed in at rewrite time? If we just set it, how should adding the destroy_map be handled?

Reply:
This is just the helper function. The rewrite creates the Op directly, it doesn't use any helper function

Reply:
The basic idea is that by default it's never inplace and then we make it inplace ourselves. The users don't need to worry about it

Reply:
Oh I get it